test/sql/local/irc/delete/delete_from_table.test

```diff
@@ -0,0 +1,62 @@
+# name: test/sql/local/irc/delete/delete_from_table.test
+# description: test integration with iceberg catalog read
+# group: [delete]
+
+require-env ICEBERG_SERVER_AVAILABLE
+
+require avro
+
+require parquet
+
+require iceberg
+
+require httpfs
+
+# Do not ignore 'HTTP' error messages!
+set ignore_error_messages
+
+statement ok
+pragma enable_logging('HTTP');
+
+statement ok
+set logging_level='debug'
+
+statement ok
+CREATE SECRET (
+    TYPE S3,
+    KEY_ID 'admin',
+    SECRET 'password',
+    ENDPOINT '127.0.0.1:9000',
+    URL_STYLE 'path',
+    USE_SSL 0
+);
+
+
+statement ok
+ATTACH '' AS my_datalake (
+    TYPE ICEBERG,
+    CLIENT_ID 'admin',
+    CLIENT_SECRET 'password',
+    ENDPOINT 'http://127.0.0.1:8181'
+);
+
+statement ok
+drop table if exists my_datalake.default.test_delete_table;
+
+statement ok
+create table my_datalake.default.test_delete_table as select range a from range(10);
```

Comment:
Can we spread this out over 5 different inserts?
I just want to verify that we can do a delete that touches multiple different data files in the same transaction